css/main.css

```diff
@@ -0,0 +1,552 @@
+    text-align: left;
+    font-size: 20px;
+    margin: 30px 0 30px 50px;
+}
+
+#box-one{
+  background-color: #555;
+}
+
+#box-two{
+  background-color: #777;
+}
+
+#box-three{
+  background-color: #333;
+}
+
+#box-four{
+  background-color: #444;
+}
+
+#main-posts{
+    padding: 0 40px;
+}
+/*ends*/
+
+/* highlited posts CSS begins*/
+.posts-items{
+    color: #000;
+    float: left;
+    width: 25%;
+    height: 500px;
+    padding: 150px 0;
+}
+
+.posts-items-big{
+    float: left;
+    width: 50%;
+    height: 650px;
+    padding: 40px 0;
+    margin-bottom: 40px;
+    border-radius: 20px;
+    background-color: #888888;
+}
+
+.posts-items-big h1 {
+    color: #fff;
+    text-decoration: underline;
+    height: 100px;
+    margin: 0 auto;
+    margin-top: 180px;
+    text-align: center;
+    padding: 10px;
+    width: 40%;
+}
+
+.posts-items-big :nth-child(2){
+
+    height: 180px;
+    margin: 0 auto;
+    margin-top: 30px;
+    width: 60%;
+}
+
+.posts-items-big p{
+
+    color: #fff;
+    height: 20px;
+    margin: 0 auto;
+    margin-top: 30px;
+    width: 40%;
+}
+
+.posts-items-big div {
+    color: #77f;
+    background-color: #fff;
+    text-align: center;
+    padding-top: 20px;
+    height: 50px;
+    margin: 0 auto;
+    margin-top: 30px;
+    width: 35%;
+    border-radius: 8px;
+}
+
+.posts-items h1 {
+    background-color: #fff;
+    height: 125px;
+    padding-left: 10px;
+    margin: 0 auto;
+    width: 75%;
+}
+
+.posts-items p{
+    background-color: #fff;
+    height: 330px;
+    margin: 0 auto;
+    margin-top: 25px;
+    padding-left: 10px;
+    font-size: 22px;
+    width: 75%;
+}
+
+.posts-items span {
+    display: block;
+    text-decoration: underline;
+    font-weight: 600;
+    margin-top: 25px;
+    margin-left: 38px;
+    color: #111;
+    height: 20px;
+    width: 50%;
+    padding-left: 10px;
+}
+
+.posts-items div {
+    margin: 0 auto;
+    margin-top: 50px;
+    background-color: #ddd;
+    height: 300px;
+    width: 80%;
+    border-radius: 10px;
+}
+/*ends*/
+
+.clear {
+    clear: both;
+  }
+
+/*Upcoming events CSS*/
+.event-box{
+  background-color: #777;
+  color: #fff;
+  height: 800px;
+  text-align: center;
+
+}
+
+#exp-spkr{
+  color: #77f;
+  background-color: #fff;
+  border-radius: 10px;
+  width: 30%;
+  margin-left: 35%;
+  height: 50px;
+  padding-top: 25px;
+  font-size: 21px;
+  font-weight: 900;
+
+}
+
+#explore{
+  padding: 20px 70px;
+  border-radius: 10px;
+  font-size: 20px;
+  font-weight: bold;
+  background-color: #eee;
+}
+/*ends*/
+
+/* Person of the Week CSS begins*/
+#person-week {
+    height: 250px;
+    background: #eee;
+    padding: 80px 80px 30px 100px;
+}
+
+#person-week .image {
+    float: left;
+    background-color: #444;
+    border-radius: 5px;
+    height: 180px;
+    width: 180px;
+}
+
+#person-week div:nth-child(2) {
+    width: 900px;
+}
+
+#person-week h3 {
+    margin-left: 15px;
+    font-weight: lighter;
+}
+
+#person-week p {
+    font-weight: 250;
```

Comment:
[Use multiples of 100](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight)